activities.json

```
@@ -687,6 +687,19 @@
     "title": "Get Installed Related Apps API",
     "url": "https://wicg.github.io/get-installed-related-apps/spec"
   },
+  {
+    "ciuName": null,
+    "description": "This spec defines a signal that conveys a person's request to websites and services to not sell or share their personal information with third parties.",
+    "id": "gpc",
+    "mdnUrl": ["https://developer.mozilla.org/en-US/docs/Web/API/Navigator/globalPrivacyControl"],
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1848951",
+    "mozPosition": "positive",
+    "mozPositionDetail": "This signal defined in this specification provides users with a way to opt-out of the disclosure of their information to third parties once per profile in a way that is legally enforced in some jurisdictions, and is being considered in future regulations.",
+    "mozPositionIssue": 867,
+    "org": "W3C",
```

I missed this, but this really needs to be a proposal.  We need to be consistent.  If this changes (and it might), then we can update.

```suggestion
    "org": "Proposal",
```